### PR TITLE
🐛 fix: da-server otel integration

### PR DESCRIPTION
**Pull request by dsun0720**
- optimize otel auto Instrumentation configure in da server
- fix the bug of migration in pro mode
- fix the bug: api and evaluation server can't start up in arm64 arch when enable otel

### CODE REVIEW

File: modules/data-analytics/start.sh
Comment (deleteLater) [inline modules/data-analytics/start.sh:1]: Prefer use `=` instead of `==`. Check https://stackoverflow.com/questions/20449543/shell-equality-operators-eq

File: modules/back-end/deploy/Dockerfile
Comment (deleteLater) [inline modules/back-end/deploy/Dockerfile:1]: Why this change?

File: modules/data-analytics/.dockerignore
Comment (deleteLater) [inline modules/data-analytics/.dockerignore:1]: Seems to no reason for this change

File: modules/data-analytics/app/health/views.py
Comment (deleteLater) [inline modules/data-analytics/app/health/views.py:1]: unused change

File: modules/data-analytics/.dockerignore
Comment (dsun0720) [reply modules/data-analytics/.dockerignore:1]: integration_tests is just used to test in local, not fit in docker image

File: modules/back-end/deploy/Dockerfile
Comment (dsun0720) [reply modules/back-end/deploy/Dockerfile:1]: api and els should be able to start in arm64. https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/2181

File: modules/data-analytics/app/health/views.py
Comment (dsun0720) [reply modules/data-analytics/app/health/views.py:1]: flake 8 format

**Review by deleteLater — APPROVED** [review]
LGTM
